//! THIS IS A TEST PROGRAM
//! COMPILER
//! FAILURE
//! CODE: 1
//! ERROR:
//! extra_arguments_in_func_call.bu:14:5: Too many arguments for Function `bla`.
//! extra_arguments_in_func_call.bu:9:1: Function `bla` expects 1 arguments, found 2.

func bla(a: i32) {

}

func main() {
    bla(1, 2);
}